docs/crds/grafana.md

```diff
@@ -0,0 +1,56 @@
+# Building Grafana Dashboard
```

Comment:
this file should be placed under docs/metrics

Reply:
changed the path of `grafana.md` and `metrics.md` to `docs/metrics`